src/app/service/paypal.js

```
@@ -1,26 +1,51 @@
 import config from '../../../config'
+import {requestJson} from './http'
 
-export function createPayment({amount, currency, offerId}) {
-  const paypal = global.paypal
+const PAYMENT_ENDPOINT = `${config.printingEngineBaseUrl}/payment/paypal`
 
-  const env = config.paypal.env
-  const client = config.paypal.client
+export function createPayment({amount, currency, offerId, shippingAddress}) {
+  const {
+    firstName,
+    lastName,
+    street,
+    houseNumber,
+    addressLine2,
+    city,
+    zipCode,
+    stateCode,
+    countryCode
+  } = shippingAddress
   const transactions = [
     {
       custom: offerId,
       amount: {
         total: String(amount),
         currency
+      },
+      item_list: {
+        shipping_address: {
+          recipient_name: `${firstName} ${lastName}`,
+          line1: `${street} ${houseNumber}`,
+          line2: addressLine2,
+          city,
+          country_code: countryCode,
+          postal_code: zipCode,
+          state: stateCode
+        }
       }
     }
   ]
-  return paypal.rest.payment.create(env, client, {transactions})
+
+  return requestJson(PAYMENT_ENDPOINT, {method: 'POST', body: {orderId: offerId, transactions}})
```

`{ orderId: offerId }` is actually wrong. After splitting the order creation and the payment process, we will have an `orderId` to be sent along when calling the `payment` creation endpoint. 

Right now it's not really an issue as our API-side check does fail silently telling us that there is no such `oderId`. In order to release without splitting things apart, we could change the API and make `orderId` optional for now.

---

`UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Not found: Order not found`

---

As discussed with @henry-all3dp we can't make `orderId` optional. That means we have to split it up in the frontend before releasing.